charts/karmada/values.yaml

```diff
@@ -768,7 +779,6 @@ search:
   ## @param search.labels labels of the search deployment
   labels:
     app: karmada-search
-    apiserver: "true"
```

Comment:
I think just keep this change is ok. The other changes don't seem to make much sense. The reason for the failure of the scheduler Estimator is that the clusterName is not filled in.

Reply:
I'm sorry, my ide plugin have a bit problem. I will modify agin.

Reply:
@Poor12 I have a test, the failure reason of scheduler Estimator due to clusterName.
but the labels is necessary to scheduler Estimator.